modules/api/src/main/java/org/apache/ignite/lang/ColumnAlreadyExistsException.java

```diff
@@ -27,6 +30,6 @@ public class ColumnAlreadyExistsException extends IgniteException {
      * @param name Column name.
      */
     public ColumnAlreadyExistsException(String name) {
```

Comment:
It's weird that exceptions like these are in "lang" module. Makes no sense. We should fix it in the future.

Reply:
I'm inclined to agree with you.